test/integration/service/serverspec/debian_spec.rb

```diff
@@ -43,6 +43,10 @@
         it { should be_grouped_into 'root' }
       end
 
+      describe file('/etc/service/plain-defaults/log/config') do
```

Comment:
If I understand correctly, svlogd's configuration file should be put into log directory like '/etc/service/plain-defaults/lig/main/config'. It should not be at the same level with log's run script.